<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>211</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Noto Sans CJK KR Black</family>
    <weight>75</weight>
    <bold>true</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <widget class="QLabel" name="title">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>20</y>
     <width>801</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Noto Sans CJK KR Black</family>
     <pointsize>26</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>SamulHAM</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="loginBtn">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>100</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Noto Sans CJK KR Black</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color : #FFFCDC;
border-radius : 5px;</string>
   </property>
   <property name="text">
    <string>로그인</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="numberInput">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>100</y>
     <width>271</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius : 5px;
background-color : white;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>-10</y>
     <width>801</width>
     <height>371</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color : #EDD2F3;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="indent">
    <number>-1</number>
   </property>
  </widget>
  <widget class="QPushButton" name="howToUseBtn">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>170</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Noto Sans CJK KR Black</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color : #516BEB;
border-radius : 5px;</string>
   </property>
   <property name="text">
    <string>사용법</string>
   </property>
  </widget>
  <widget class="QPushButton" name="creditBtn">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>170</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Noto Sans CJK KR Black</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color : #516BEB;
border-radius : 5px;</string>
   </property>
   <property name="text">
    <string>크레딧</string>
   </property>
  </widget>
  <zorder>label</zorder>
  <zorder>title</zorder>
  <zorder>loginBtn</zorder>
  <zorder>numberInput</zorder>
  <zorder>howToUseBtn</zorder>
  <zorder>creditBtn</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
